benchmarks/animation-bp/app.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+angular.
+  module('animationBenchmark', ['ngAnimate'], config).
```

Comment:
I dislike this dot placement a lot. :P We use the leading dot in most places and I'd like to ESLint-enforce it one day...